src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -46,6 +46,7 @@ import ManifestDetailSidePanel from '../ManifestDetailSidePanel';
 import './SatelliteManifestPanel.scss';
 import DeleteManifestConfirmationModal from '../DeleteManifestConfirmationModal';
 import SCAStatusSwitch from '../SCAStatusSwitch';
+import { updateExpressionWithTypeArguments } from 'typescript';
```

Comment:
This can be removed if it's not being used

"NoManifestFound" being imported doesn't seem to be used either